### DISCUSSION TITLE
样例5中可以传pdf，为什么我下面代码中会报

### DISCUSSION THREAD

**Discussion by ZGuangW** [PaddlePaddle/PaddleOCR#12464] [Q&A]
[样例5](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_ch/whl.md#5-pdf%E6%96%87%E4%BB%B6%E4%BD%9C%E4%B8%BA%E8%BE%93%E5%85%A5)中可以传pdf，为什么我下面代码中result = ocr.ocr(pdf_file, cls=True)会报'Document' object has no attribute 'pageCount'

```python
# 导入所需库
from paddleocr import PaddleOCR

# 初始化PaddleOCR
ocr = PaddleOCR(use_angle_cls=True, lang="ch", page_num=0)

# 读取PDF文件
pdf_file = r"D:\Desktop\测试.pdf"

# 使用PaddleOCR识别PDF中的文本
result = ocr.ocr(pdf_file, cls=True)

# 输出识别结果
for line in result:
    print(line[-1][0])
```

**Comment by xxcoco763** [comment]
版本问题，按照报错提示，改一下代码就好了。除了page_count，还有其他的几个，同样按照报错提示改就好

**Comment by javachens** [comment]
> 版本问题，按照报错提示，改一下代码就好了。除了page_count，还有其他的几个，同样按照报错提示改就好

@xxcoco763 我改了这些，虽然没报错了，但是加载pdf失败
[2024/03/29 10:00:08] ppocr ERROR: error in loading image:./imgs/22.pdf
